packages/env-spec-parser/test/functions.test.ts

```diff
@@ -135,6 +146,51 @@ describe('ref expansion', functionValueTests({
   },
 }));
 
+describe('replace expansion', functionValueTests({
```

Comment:
No need for this - as long as the ref stuff is tested and replace is tested, we can assume they will work correctly together.